selectBoard.php

```diff
@@ -0,0 +1,211 @@
+    <link href="CSS/BoardView-Body.css" rel="stylesheet">
+    <title>Select Board</title>
+
+</head>
+<div class="jumbotron">
+        <div class="container">
+            <h2 class="display-5">Welcome <?php echo $_SESSION['user']; ?></h2>
+            <a href="logout.php" class="btn btn-warning">Logout</a>
+        </div>
+</div>
+
+
+    <!-- button to create board -->
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#createBoard">
+    Create Board
+    </button>
+
+
+    
+
+<div class="row">
+<?php
+$stmt = $db->prepare('SELECT * FROM share WHERE `u.WATIAM` = ? ORDER BY `b.boardID` DESC');
+            $stmt->execute(array($_SESSION['user']));       
+            $shares = $stmt->fetchAll();
+foreach ($shares as $share){
+	$stmt = $db->prepare('SELECT * FROM board WHERE `boardID` = ? ORDER BY `boardID` DESC');
+            $stmt->execute(array($share['b.boardID']));       
+            $board = $stmt->fetch();
+?>
+<div class='col-xs-2'>
+	<div class="card">
+	  <div class="card-body">
+	    <h5 class="card-title">Board: <?php echo $board['boardTitle']; ?></h5>
+	    <h6 class="card-subtitle mb-2 text-muted"><?php echo $share['permission']; ?> access</h6>
+	    <p class="card-text">Created <?php echo date("Y-m-d H:i:s", $board['boardDateCreated']); ?> by <?php echo $board['u.WATIAM']; ?></p>
+		<div>   
+		    <a href="listView.php?board=<?php echo $board['boardID']; ?>" class="btn btn-primary" style="display:inline-block;">View Board</a>
+		    <?php if ($share['permission'] == "edit") {
+		   	?>
+		   	<button type="button" class="btn btn-info" data-toggle="modal" data-target="#shareBoard">
+		    Share
+		    </button>
+		   	<form action = "selectBoard.php" method = "POST" style="display:inline-block;"> 
+		      <input type="hidden" name="deleteID" value="<?php echo $board['boardID']; ?>">
+		      <button type="submit" class="btn btn-danger" style="display:inline-block;">
+		      Delete Board
+		      </button>
+		    </form>
+		    
+		   	<?php
+		    }
+		    ?>
+		    
+		</div> 
+	  </div>
+	</div>
+</div>
+<div class="modal fade" id="shareBoard" tabindex="-1" aria-labelledby="shareBoard" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLabel">Share Board</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <form action="" class="edit-ListName" method="POST">
+        	<?php
+        	$stmt = $db->prepare('SELECT * FROM user ORDER BY `WATIAM` ASC');
+            $stmt->execute();       
+            $users = $stmt->fetchAll();
+        	?>
+            <select name="user" class="form-control" required>
+            <option value="" disabled selected>Select User</option>
+            <?php foreach($users as $user): ?>
```

Comment:
While the dropdown of every user works great for our current prototype, we should consider adding by username in the future as a large userbase would make this incredibly burdensome